projects/plugins/crm/src/automation/commons/triggers/events/class-event-deleted.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Jetpack CRM Automation Event_Deleted trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Event_Deleted class.
+ */
+class Event_Deleted extends Base_Trigger {
+
+	/**
+	 * Get the slug name of the trigger.
+	 * @return string
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/event_deleted';
+	}
+
+	/**
+	 * Get the title of the trigger.
+	 * @return string
+	 */
+	public static function get_title(): ?string {
+		return __( 'Delete Event', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the description of the trigger.
+	 * @return string
+	 */
+	public static function get_description(): ?string {
+		return __( 'Triggered when a event is deleted', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the category of the trigger.
+	 * @return string
+	 */
+	public static function get_category(): ?string {
+		return __( 'event', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Listen to this trigger's target event.
+	 */
+	protected function listen_to_event() {
+		add_action(
+			'jpcrm_event_delete',
```

Comment:
I can't seem to find this hook in the codebase. Is it just me? Or is it perhaps part of the event project Clemente is working on? 🙈

Reply:
I'm following the pattern as explained here: p1690291864361459/1690291362.861599-slack-CTXBP902X